lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -14,140 +14,159 @@ import LoggedInUser from './logged_in_user';
 import LoggedOutUser from './logged_out_user';
 import Success from './success';
 
+const initialFormErrors = {
+  course: undefined,
+  subject: undefined,
+  message: undefined,
+  request: undefined,
+};
 
 class RenderForm extends React.Component {
   constructor(props) {
     super(props);
+    this.submitFormUrl = this.props.context.submitFormUrl;
     this.userInformation = this.props.context.user;
     const course = this.userInformation ? this.userInformation.course_id : '';
+    this.courseDiscussionURL = '/courses/{course_id}/discussion/forum';
     this.state = {
       currentRequest: null,
-      errorList: [],
+      errorList: initialFormErrors,
       success: false,
       formData: {
+        course,
         subject: '',
         message: '',
-        course,
       },
     };
+    this.formValidationErrors = {
+      course: gettext('Select a course or select "Not specific to a course" for your support request.'),
+      subject: gettext('Select a subject for your support request.'),
+      message: gettext('Enter some details for your support request.'),
+      request: gettext('Something went wrong. Please try again later.'),
+    };
     this.submitForm = this.submitForm.bind(this);
```

Comment:
These bindings can also be removed by using arrow functions.

Reply:
Sticking to `.bind` for now.